File: src/gatling/scala/org/openapitools/client/model/OfferContentErrorDTO.scala

package org.openapitools.client.model


case class OfferContentErrorDTO (
    _type: OfferContentErrorType,
    /* Идентификатор характеристики, с которой связана ошибка. */
    _parameterId: Option[Long],
    /* Текст ошибки. */
    _message: String
)
object OfferContentErrorDTO {
    def toStringBody(var_type: Object, var_parameterId: Object, var_message: Object) =
        s"""
        | {
        | "type":$var_type,"parameterId":$var_parameterId,"message":$var_message
        | }
        """.stripMargin
}

File: ympa_fsharp_giraffe_server/src/model/OfferContentErrorDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.OfferContentErrorType

module OfferContentErrorDTO =

  //#region OfferContentErrorDTO


  type OfferContentErrorDTO = {
    Type : OfferContentErrorType;
    ParameterId : int64;
    Message : string;
  }
  //#endregion
